.env

```diff
@@ -0,0 +1,5 @@
+SID = "AC4f30143cf5eccea4c0f4cb53848159b0"
+TOKEN = "dafdda535cb4838d43f2fd45e9988591"
+
+MY_NUMBER = "+447761719320",
```

Comment:
hmm your .env file shouldn't be in github! Ooops! You should probably remove this.